qemu/tests/secure_img.py

```diff
@@ -0,0 +1,74 @@
+import logging
```

Comment:
As discussed offline, `logging` is no longer used, can use `test.log` instead. And I think can refer to `boot.py` as a template.

Reply:
sorry for the misunderstanding before, updated now